cassandra/src/main/scala/filodb.cassandra/columnstore/CassandraColumnStore.scala

```diff
@@ -155,15 +155,23 @@ extends CachedMergingColumnStore with StrictLogging {
     val metadata = clusterConnector.session.getCluster.getMetadata
     val splitsPerNode = params.getOrElse("splits_per_node", "1").toInt
     require(splitsPerNode >= 1, s"Must specify at least 1 splits_per_node, got $splitsPerNode")
-
-    val tokenRanges = metadata.getTokenRanges.asScala.toSeq.flatMap { range =>
-      range.splitEvenly(splitsPerNode).asScala
+    val tokensByReplica = metadata.getTokenRanges.asScala.toSeq.groupBy(tokenRange =>
```

Comment:
(nit/style) use `groupBy { tokenRange =>`  always use braces for multiline functions